api/bin/chainflip-ingress-egress-tracker/README.md

```diff
@@ -1,13 +1,17 @@
 # About
 
-Ingress-Egress Tracker observes events on external blockchains (ETH, DOT, BTC) and provides a way for client applications to subscribe and receive
-these events via a WebSocket subscription. For BTC, the tracker exposes a separate RPC call to query transactions in the mempool in addition to the
-WebSocket subscription.
+Ingress-Egress Tracker observes events on external blockchains (ETH, DOT, BTC)
+and deposit information for deposit addresses and broadcast information into a
+Redis database. For BTC, the tracker also inserts information about mempool
+transactions into Redis.
 
 # Setup
 
-The tracker will start an RPC server on 0.0.0.0:13337 (at the moment this cannot be configured).
-When working with a "localnet" (e.g. for development purposes), no extra configuration is necessary: `./chainflip-ingress-egress-tracker`.
+To start a Redis database locally, run `docker-compose up -d redis`.
+
+When working with a "localnet" (e.g. for development purposes), no extra
+configuration is necessary: `./chainflip-ingress-egress-tracker`.
```

Comment:
Created this: https://linear.app/chainflip/issue/PRO-1099/start-ingress-egress-tracker-on-localnet-startup